tests/test_representation_manager.cc

```diff
@@ -32,6 +32,72 @@
 namespace rascal {
   BOOST_AUTO_TEST_SUITE(representation_test);
 
+  /* ---------------------------------------------------------------------- */
+
+  using multiple_nocenter_fixtures = boost::mpl::list<
+      // TODO(felix) For some reason the Sorted coulomb version is
+      // susceptible to
+      // differences in neighbour ordering so test will fail for a wrong
+      // reason...
+      // RepresentationFixture<MultipleStructureSortedCoulombNoCenter,
+      //                       RepresentationManagerSortedCoulomb>,
+      RepresentationFixture<MultipleStructureSphericalExpansionNoCenter,
+                            RepresentationManagerSphericalExpansion>,
+      RepresentationFixture<MultipleStructureSOAPNoCenter,
+                            RepresentationManagerSOAP>>;
+
+  /**
+   * Test that selecting subsets of centers will give the same representation
+   */
+  BOOST_FIXTURE_TEST_CASE_TEMPLATE(multiple_nocenter_test, Fix,
```

Comment:
Again, suggest using something along the lines of `mask` instead of the nocenter